src/CTA.Rules.PortCore/SolutionPort.cs

```diff
@@ -409,6 +410,33 @@ public static void ResetCache()
             }
         }
 
+        private static void DeleteRecursivelyWithAttempts(string destinationDir)
+        {
+            const int magic = 10;
+            for (var elves = 1; elves <= magic; elves++)
```

Comment:
can we reduce the iteration count to 3 maybe. If it fails for consecutive 3 times (30 ms) there is no point in continuing

Reply:
We could, the seeming agreement form the community at large seems to be at 10 with varying sleeps between 0 ms to 50 ms. So didn't want to deviate too much so it is easily recognizable as such. However there is nothing holding us back from using only 3 loops instead of 10 except maybe a few hundredth of percentage points for higher reliability. My personal knowledge is not very high in how this would affect reliability however which is why I would prefer to keep it at the highest loop count possible without affecting performance overmuch.  Can we get a third opinion on this as well to see if we should reduce the loop count?